crypto/src/hash/rescue_prime/mod.rs

```diff
@@ -0,0 +1,816 @@
+use crate::alloc::vec::Vec;
+use core::iter;
+use lambdaworks_math::field::element::FieldElement;
+use lambdaworks_math::field::fields::u64_goldilocks_field::Goldilocks64Field;
+
+mod parameters;
+mod utils;
+
+use parameters::*;
+use utils::*;
+
+type Fp = FieldElement<Goldilocks64Field>;
+
+#[derive(Clone)]
+#[allow(dead_code)]
+enum MdsMethod {
+    MatrixMultiplication,
+    Ntt,
+    Karatsuba,
+}
+#[warn(dead_code)]
```

Comment:
why do you need this annotation?

Reply:
Solved, it wasn’t necessary. I added it while trying to fix some Clippy issues, but it turned out to be incorrect